pilz_extensions/include/pilz_extensions/joint_limits_interface_extension.h

```diff
@@ -52,7 +52,7 @@ inline bool getJointLimits(const std::string& joint_name,
 
   // Set the existing limits
   if(! ::joint_limits_interface::getJointLimits(joint_name, nh, limits)) {
-    return false;
+    return false; //LCOV_EXCL_LINE // The case where getJointLimits returns false is covered above.
```

Comment:
Then why is the check needed here?

Reply:
Because the behaviour of the external `getJointLimits` might change. Then, without this check, reading the joint limits could be omitted silently.